src/app/(nextjs_migration)/(guest)/page.tsx

```diff
@@ -0,0 +1,152 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Image from "next/image";
+import Script from "next/script";
+import "../../../client/css/partials/landing.css";
+
+import { getL10n } from "../../functions/server/l10n";
+
+import HeroImage from "../../../client/images/landing-hero@2x.webp";
+import LaptopImage from "../../../client/images/landing-laptop@2x.webp";
+import LockImage from "../../../client/images/landing-lock@2x.webp";
+import MailImage from "../../../client/images/landing-mail@2x.webp";
+import NaturePhoneImage from "../../../client/images/landing-nature-phone@2x.webp";
```

Comment:
Open question: We’ll have to agree on how to handle responsive images best.

Reply:
I think [the <Image> component](https://nextjs.org/docs/app/api-reference/components/image#src) should handle that?
> Asset Flexibility: On-demand image resizing, even for images stored on remote servers